### ISSUE TITLE
Mac with M1 Chip support

### ISSUE THREAD

**Issue by alexzy-LFS** [HelixNGC7293/DeforumStableDiffusionLocal#2]
Hi, would you support this package on Mac with M1 chips? thanks

**Comment by HelixNGC7293** [comment]
Hello ~ I haven't tested it on Mac since I'm not very familiar with M1 structure as well, but I'll check it out in the future to see if a mac compatible version can be made ;)

**Comment by LandonReillyTroy** [comment]
Yes this would be awesome, it looks like the code isn't targeting 'mps' for apple silicon.
